core/src/main/java/org/sonarsource/sonarlint/core/StandaloneSonarLintEngineImpl.java

```diff
@@ -80,20 +80,15 @@ public RuleDetails getRuleDetails(String ruleKey) {
   }
 
   @Override
-  public AnalysisResults analyze(StandaloneAnalysisConfiguration configuration, IssueListener issueListener) {
-    return analyze(configuration, issueListener, null);
```

Comment:
This is mainly to keep a simple interface, without having multiple overloaded methods. Also it forces clients to update and decide what to do.